geometry_msgs/Point32[] P
geometry_msgs/Point32 P0
geometry_msgs/Point32 n
float32 epsilon
---
geometry_msgs/Point32[] P
